### ISSUE TITLE
Unable to create Timeboards

### ISSUE THREAD

**Issue by jo-tez** [mmamaev/timeboard#9]
```
import timeboard.calendars.UK as UK
clnd = UK.Weekly8x5()
```

Produces error: _TypeError: __new__() got an unexpected keyword argument start_
There doesn't seem to be a way of creating timeboards, either through the calendar or directly

**Comment by btmartin721** [comment]
Having the same exact issue. I tried using US.Weekly8x5() and tb.Timeboard using the examples provided in the user guide. It always gives me one of two errors:

```TypeError: __new__() got an unexpected keyword argument start```

or 

```TypeError: __new__() got an unexpected keyword argument end```

Here's the code I used: 

```
import timeboard as tb
weekly = tb.Organizer(marker='W', structure=[[1,1,1,1,1,0,0]])	
	clnd = tb.Timeboard(base_unit_freq='D', start='01 Oct 2017', end='12 Oct 2017', layout=weekly)
```

I've tried using Python 3.6 and 2.7 with no success. I think something in the package is broken.

**Comment by lotusflyer** [comment]
same problem here :-(

**Comment by mmamaev** [comment]
Sorry guys, this is probably pandas that have changed their API.  I will look into it soon. Meanwhile try to downgrade  pandas to 0.24.

**Comment by btmartin721** [comment]
Downgrading to pandas 0.24.2 worked for me. It now runs, though it displays some deprecation warnings.

Thanks for that suggestion.

-Bradley

**Comment by elpaisik** [comment]
I have the same problem. The first issue is that the pandas.PeriodIndex class no longer supports 'start', 'periods' and 'end' parameters (see core.py line 227). Instead, pandas.period_range() should be used. I tried to fix this problem but subsequent errors occured which I was not able to fix.

**Comment by mmamaev** [comment]
Fixed in timeboard 0.2.3. Now it is compatible with pandas 1.0.

**Comment by elpaisik** [comment]
> Fixed in timeboard 0.2.3. Now it is compatible with pandas 1.0.

Thanks a lot, it now works for me. As far as I can tell: Great Job!

**Comment by mmamaev** [comment]
You're welcome :)
